src/rsz/src/Resizer.i

```diff
@@ -745,6 +753,14 @@ set_worst_slack_nets_percent(float percent)
   resizer->setWorstSlackNetsPercent(percent);
 }
 
+void
+synthesize_buffers_cmd(int max_fanout, float gain, float slew)
```

Comment:
How would a user choose gain and slew?

Reply:
That's TBD. Once we confirm this algorithm works I would like for us to experiment a bit with other algorithms which may need less user input